HELM-368: Update HELM docs (#624)
OpenNMS/opennms-helm: docs/modules/panel_configuration/pages/forecasting.adoc

** Your time range must include at least more than twice the period size (in this case, 7200 seconds or more) to retrieve the historical data required for the forecast.
** Consider the seasonality of your data when specifying a time range (for example, you may not want to include weekend network usage in a forecast of traffic during regular business hours).
** Note that if your data is noisy or spiky, forecasting does not work as well.
In these cases, you should add the `Chomp` and `Outlier` filters and rearrange them so they appear before the `HoltWinters` filter in the list of queries.
You must specify an input for the `Outlier` filter.
. Click *Save*.

=== Display forecast data

Because the default time range for a panel returns data from the past to the present, you must create a custom time range to display forecast data:

. Select *Custom Time Range* from the *Time Range* list.
. Specify the start of your time range in the *From* box (for example, `now-6h`).
. Specify the end of your time range in the *To* box:
+
image::pc-time-range.png["Example dashboard time range configuration.", 400]

The panel displays the forecast data for the time period that you specify:

> +

Author: ```suggestion

```

> image::pc-forecast.png["Example panel displaying forecast data.", 700]

Reviewer: This image did not display.

Author: Odd. I'll double-check on my end, but I believe it rendered without issue when I built locally.

Reviewer: Ya, it's strange. All the other images rendered for me, and I looked that the syntax, and it seems fine.

You can experiment with the time range function to see how it updates your forecast results.
Note that the further into the future you forecast, the less authoritative the data will be.

== Filters

> The following predefined filters can be used to create forecasting metrics:

Reviewer: ```suggestion
You can use the following predefined filters to create forecasting metrics:
```

> [options="autowidth"]

Reviewer: The filter column is too narrow, so the actual filter names are getting smushed, even at full screen.

Author: ```suggestion
[cols="1,2,2"]
```

|===
| Filter    | Description   | Use

| Chomp
| Strips leading and trailing rows that contain only `NaN`/`null` values.

> | This filter is useful when the values from the data source do not cover the entire time interval used by the report.

Reviewer: ```suggestion
| Useful when the values from the data source do not cover the entire time interval the report uses.
```
